### DISCUSSION TITLE
二月更新

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#756] [通知]
## 🚀 功能

- 添加返回主页按钮 @Richasy (#755)
- 支持清理本地图片缓存 @Richasy (#752)

## 🐛 Bug 修复

- Fix ARM64 platform QRCode error. @zhzy0077 (#754)
- 修复热搜Id解析问题 @Richasy (#753)
- 修复视频信息没有设置成功的问题 #684 @artiga033 (#721)

## 🧰 管理与维护

- 更新 Issue 模板 @Richasy (#712)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2202.1.0'>二月更新</a>.</em>

**Comment by xyl23333** [comment]
第一

**Reply by DieMoon98** [reply]
第二

**Reply by Alpha-lab1** [reply]
第三

**Reply by Michael66625** [reply]
第四

**Reply by Zero-wyc** [reply]
> 第一
第五

**Comment by YanSu5001** [comment]
![image](https://user-images.githubusercontent.com/90969621/153710189-00ce2dd5-f9ce-4a5f-9b61-2be49fa0c25f.png)

**Reply by YanSu5001** [reply]
咱就是说作为uwp，这个界面能不能好好优化一下，哈哈哈哈哈。像右边爱奇艺这样这样就做的很不错。

**Reply by Richasy** [reply]
你再把应用宽度变窄点或者拉宽点效果都不算差，屏幕宽度足够，没必要把应用弄得这么窄

**Reply by YanSu5001** [reply]
> 你再把应用宽度变窄点或者拉宽点效果都不算差，屏幕宽度足够，没必要把应用弄得这么窄

我明白了，确实再稍微拉窄一点就如爱奇艺uwp的显示效果了。

**Reply by ViewWay** [reply]
有的用就不错了
